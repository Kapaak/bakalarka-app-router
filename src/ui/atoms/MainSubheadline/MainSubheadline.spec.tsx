import { render } from "@testing-library/react";

import { MainSubheadline } from "./MainSubheadline";

describe("MainSubheadline", () => {
  it("should render successfully", () => {
    const { baseElement } = render(<MainSubheadline />);

    expect(baseElement).toBeTruthy();
    expect(baseElement).toMatchSnapshot();
  });
});
