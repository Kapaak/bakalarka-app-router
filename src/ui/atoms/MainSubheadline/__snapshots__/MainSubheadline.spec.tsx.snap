// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainSubheadline should render successfully 1`] = `
<body>
  <div>
    <div
      class="text-2xl lg:mt-2 undefined"
    />
  </div>
</body>
`;
